com.unity.multiplayer.mlapi/Runtime/Core/SnapshotRTT.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using UnityEngine;
+
+namespace MLAPI
+{
+    internal class ClientRtt
+    {
+        /// <summary>
+        /// Round-trip-time data
+        /// </summary>
+        public struct Rtt
+        {
+            public double Best;
+            public double Average;
+            public double Worst;
+            public int SampleCount;
+        }
```

Comment:
+ Last maybe?
also this struct could be outside of `ClientRtt` class.

Reply:
I'd also add unit suffixes (assuming they're seconds): `BestSec`, `AverageSec`, `WorstSec`.